static/main.html

```diff
@@ -16,6 +16,7 @@
     <link href="../node_modules/@blueprintjs/core/lib/css/blueprint.css" rel="stylesheet" />
     <!-- add other blueprint-*.css files here -->
     <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="conversations.css">
```

Comment:
Note that we used to do `{ prepend: true }` with `insert-css` module. We get some bug with scrolling otherwise:

![screenshot from 2019-01-09 23-58-41](https://user-images.githubusercontent.com/308049/50935082-18b3bf00-146b-11e9-97dc-1714a0ab126f.png)